test/unit/Platform/TargetPhp/PhpBinaryPathTest.php

```diff
@@ -135,6 +135,10 @@ public static function phpConfigPathProvider(): array
                 && is_executable($phpConfigPath[0]),
         );
 
+        if ($possiblePhpConfigPaths === []) {
+            return ['skip' => ['skip', 'skip']];
```

Comment:
Wouldn't this still try to run a test; you would need to add a check for `skip` into `testFromPhpConfigExecutable`; the windows skip above only works because the `testFromPhpConfigExecutable` test has the exact same check. That check could maybe be changed to see `if ($phpConfigPath === 'skip') { self::markTestSkipped...` perhaps?